apps/crn-server/src/controllers/tutorial.controller.ts

```diff
@@ -9,6 +13,15 @@ export default class TutorialController {
     this.tutorialDataProvider = tutorialsDataProvider;
   }
 
+  async fetch(options: FetchOptions): Promise<ListTutorialsResponse> {
+    const { total, items } = await this.tutorialDataProvider.fetch(options);
+
+    return {
+      total,
+      items,
+    };
```

Comment:
As we are not processing the result of the dataprovider call we can just return the call

```suggestion
   return  this.tutorialDataProvider.fetch(options);
```